tests/wicked/before_test.pm

```diff
@@ -53,6 +53,7 @@ sub run {
         systemctl 'start dhcpd.service';
     } else {
         # Common SUT Configuration
+        $self->get_from_data('wicked/ifbind.sh', 'ifbind.sh', executable => 1);
```

Comment:
what if 
```suggestion
        $self->get_from_data('wicked/ifbind.sh', '/ifbind.sh', executable => 1);
```

in this case it would simplify place where you calling it